src/network/networkd-gperf.gperf

```diff
@@ -30,8 +30,10 @@ Network.RouteTable,                      config_parse_route_table_names,
 Network.IPv4Forwarding,                  config_parse_tristate,                  0,          offsetof(Manager, ip_forwarding[0])
 Network.IPv6Forwarding,                  config_parse_tristate,                  0,          offsetof(Manager, ip_forwarding[1])
 Network.IPv6PrivacyExtensions,           config_parse_ipv6_privacy_extensions,   0,          offsetof(Manager, ipv6_privacy_extensions)
+DHCPv4.UseDomains,                       config_parse_default_dhcp_use_domains,  0,          offsetof(Manager, dhcp_use_domains)
```

Comment:
Personally I would use `DefaultUseDomains` here, as `system.conf` does for most of its default parameters.

Reply:
I started with _`_DefaultUseDomains_`_ but @yuwata raised a point where having a separate naming might be redundant (including the existing configs). Perhaps something we can further discuss here to determine what the go-to pattern should look like?

Reply:
I agree with @yuwata that `UseDomains=` should be the way to go. Not only because `Default` is redundant but also that `networkd.conf` is already using such naming scheme.